src/pyfmi/fmi.pyx

```diff
@@ -1805,17 +1814,21 @@ cdef class FMUModelBase(ModelBase):
 
         Calls the low-level FMI function: fmiGetString
         """
-        cdef int         status
+        cdef int status
         cdef N.ndarray[FMIL.fmi1_value_reference_t, ndim=1, mode='c'] input_valueref = N.array(valueref, dtype=N.uint32, ndmin=1).ravel()
-        cdef FMIL.fmi1_string_t* output_value = <FMIL.fmi1_string_t*>FMIL.malloc(sizeof(FMIL.fmi1_string_t)*input_valueref.size)
+        cdef int nref = input_valueref.size
```

Comment:
nref should be of type size_t:
`cdef FMIL.size_t nref`

Reply:
Fixed, also the others.